.env.example

```
@@ -11,8 +11,14 @@ BETTER_AUTH_URL=""
+# AWS S3 Configuration (필수 사항)
 AWS_ACCESS_KEY_ID="" # 보안 자격 증명 > 액세스 키 만들기
 AWS_SECRET_ACCESS_KEY=""
 AWS_REGION="ap-northeast-2"
-AWS_S3_BUCKET=""
\ No newline at end of file
+AWS_S3_BUCKET_DOCUMENTS=""
+AWS_S3_BUCKET_REPORTS=""
+# CloudFront Configuration (선택 사항)  cloudfront > 
+AWS_CLOUDFRONT_URL_DOCUMENTS="" # CloudFront 배포 도메인 이름 (예: d123.cloudfront.net)
+AWS_CLOUDFRONT_URL_REPORTS="" 
+AWS_CLOUDFRONT_KEY_PAIR_ID="" # CloudFront 키페어 ID (SIGNED URL용)
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)